main.go

```diff
@@ -27,6 +29,8 @@ func main() {
 	http.HandleFunc("/api/", httpUnauthorized)
 	http.HandleFunc("/auth/token", httpBad)
 	http.HandleFunc("/observer/logs", httpLogs)
+	http.HandleFunc("/supervisor/supervisor/logs", httpSupervisorProxy)
```

Comment:
Keep in mind that these logs are not available during Supervisor startup/setup. In practice, on a new system where the Supervisor update itself, it might be that during quite a while you won't be able to get logs.

Technically, there could be the situation where during this Supervisor update time something bad happens. So we have to still consider the observer logs too in frontend :cry: